source/_integrations/voip.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: VoIP
+description: Voice over IP
+ha_category:
+  - Voice
+ha_iot_class: Local Push
+ha_release: '2023.5'
+ha_codeowners:
+  - '@balloob'
+  - '@synesthesiam'
+ha_domain: voip
+ha_integration_type: integration
+ha_quality_scale: internal
+---
+
+The VoIP integration enables users to talk to [Assist](/docs/assist) using an analog phone and a VoIP adapter such as the [Grandstream HT801](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht801). See [the tutorial](/projects/private-voice-assistant/voice_remote_classic_phone/) for detailed instructions.
+
+<p class='img'>
+  <img src="/images/integrations/voip/voip_adapter.png" />
+  Connecting a phone to Home Assistant requires an adapter.
+</p>
+
```

Comment:
Remove all extra images in this pr that are unused

Reply:
Removed